src/deluge/gui/views/instrument_clip_view.cpp

```diff
@@ -2074,7 +2074,7 @@ void InstrumentClipView::adjustProbability(int32_t offset) {
 
 					// Incrementing
 					if (offset == 1) {
-						if (probabilityValue < kNumProbabilityValues + 35) {
+						if (probabilityValue < kNumProbabilityValues + kNumIterationValues) {
```

Comment:
Probability value represents fill, a probabilty, or an iteration right? I'd suggest renaming it to triggerCondition or something to make it less misleading

Reply:
Yes, seems like probabilityValue was named before the iteration feature was introduced. Iteration values are a separate range but all the probability values and iteration values are referenced in one long series.

I like the idea of trigger Condition but it feels like quite a big change to rename it. It's not just renaming the variable name, it would also mean possibly changing lots more function names and code comments to get the clarity to want. The word 'probability' seems to be used interchangeably for either actual percentage probability or the full range of trigger conditions.

Is this the right time to make that change?

Reply:
No lets do it in a separate PR for safety